src/main/java/com/shrek/task00001/Main.java

```diff
@@ -1,18 +1,64 @@
 package com.shrek.task00001;
 
-import sun.reflect.generics.reflectiveObjects.NotImplementedException;
 
 /**
  * Группа людей участвует в марафоне, их имена и время за которое они пробежали марафон представленны в массиве.
  * Необходимо найти человека, который быстрее всех пробежал дистанцию и вывести его имя.
- * Найти человека, который прибежал вторым.
+ * Найти  человека, который прибежал вторым.
  */
 public class Main {
     public String getWinner(String[] runners, int[] times) {
-        throw new NotImplementedException();
+
+        if (times.length == 0) {
+            return null;
+        }
+        int min1 = times[0];
+        int winnerPos = 0;
+        for (int i = 0; i <= times.length - 1; i++) {
+            if (times[i] < min1) {
+                min1 = times[i];
+            }
+            if (times[i] == min1) {
+                winnerPos = i;
+            }
+        }
+        return runners[winnerPos];
     }
 
     public String getSecond(String[] runners, int[] times) {
-        throw new NotImplementedException();
+        if (times.length < 2) {
+            return null;
+        }
+        int winnerTime = times[0];
+        int secondTime = times[1];
+
+        int secondPos = 0;
```

Comment:
It is strange that at start `secondTime` is with index `1` but `secondPos` is `0`

Reply:
Oh. I see. it is also just too broad from usage. Please use variables closed to usage